### PR TITLE
Allow mpTelemetry-2.0 to work with Java 8

### PR DESCRIPTION
**Pull request by Yas-Franco**
Allow mpTelemetry-2.0 to work with Java 8

### CODE REVIEW

**Comment by Yas-Franco** [comment]
#build

**Comment by Yas-Franco** [comment]
#libby

**Comment by yasmin-aumeeruddy** [comment]
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

File: dev/build.image/scripts/Liberty- server remote debug.launch [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (benjamin-confino) [inline dev/build.image/scripts/Liberty- server remote debug.launch]: Did you mean to include this and the other remote debug lunch file?

File: dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/suite/FATSuite.java [hunk 2]
```diff
@@ -40,7 +39,7 @@
 
 @RunWith(Suite.class)
 @SuiteClasses({
-                AlwaysPassesTest.class, //Must keep this test to run something in the Java 6 builds.
```
Comment (benjamin-confino) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/suite/FATSuite.java]: Did you mean to keep all these fats commented out?

File: dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/CrossFeatureJaegerTest.java [hunk 3]
```diff
@@ -155,12 +154,14 @@ public void testCrossFeatureFromTelemetry() throws Exception {
             Log.info(c, "testCrossFeatureFromTelemetry", span.toString());
         }
         //OpenTracing, MpTelemetry-1.0 and MpTelemetry-1.1 use HTTP_URL while MpTelemetry 2.0 uses URL_FULL
-        if (RepeatTestFilter.isRepeatActionActive(TelemetryActions.MP14_MPTEL20_ID) || RepeatTestFilter.isRepeatActionActive(TelemetryActions.MP41_MPTEL20_ID) || RepeatTestFilter.isRepeatActionActive(TelemetryActions.MP50_MPTEL20_ID) || RepeatTestFilter.isRepeatActionActive(MicroProfileActions.MP70_EE11_ID) || RepeatTestFilter.isRepeatActionActive(MicroProfileActions.MP70_EE10_ID)) {
+        if (RepeatTestFilter.isRepeatActionActive(TelemetryActions.MP14_MPTEL20_ID) || RepeatTestFilter.isRepeatActionActive(TelemetryActions.MP41_MPTEL20_ID)
```
Comment (benjamin-confino) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/CrossFeatureJaegerTest.java:157]: Is the comment above this if statement right? It implies that we should have one if for MPTel 2, and an else for 1/1.1 

but I see  all three as possibiities.

File: dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/CrossFeatureJaegerTest.java [hunk 4]
```diff
@@ -205,15 +206,17 @@ public void testCrossFeatureFromOpenTracing() throws Exception {
             Log.info(c, "testCrossFeatureFromOpenTracing", span.toString());
         }
         //OpenTracing, MpTelemetry-1.0 and MpTelemetry-1.1 use HTTP_URL while MpTelemetry 2.0 uses URL_FULL
-        if (RepeatTestFilter.isRepeatActionActive(TelemetryActions.MP14_MPTEL20_ID) || RepeatTestFilter.isRepeatActionActive(TelemetryActions.MP41_MPTEL20_ID) || RepeatTestFilter.isRepeatActionActive(TelemetryActions.MP50_MPTEL20_ID) || RepeatTestFilter.isRepeatActionActive(MicroProfileActions.MP70_EE11_ID) || RepeatTestFilter.isRepeatActionActive(MicroProfileActions.MP70_EE10_ID)) {
+        if (RepeatTestFilter.isRepeatActionActive(TelemetryActions.MP14_MPTEL20_ID) || RepeatTestFilter.isRepeatActionActive(TelemetryActions.MP41_MPTEL20_ID)
```
Comment (benjamin-confino) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/CrossFeatureJaegerTest.java:209]: Is the comment above this if statement right? It implies that we should have one if for MPTel 2, and an else for 1/1.1

but I see all three as possibiities.

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/.classpath [hunk 5]
```diff
@@ -2,6 +2,10 @@
 <classpath>
 	<classpathentry kind="src" path="fat/src"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
+		<attributes>
+			<attribute name="module" value="true"/>
```
Comment (benjamin-confino) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/.classpath:7]: Do we want module?

**Review by benjamin-confino — CHANGES_REQUESTED** [review]

File: dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/CrossFeatureJaegerTest.java (hunk 4 above)
Comment (Yas-Franco) [reply dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/CrossFeatureJaegerTest.java:209]: We have an if for 1/1.1 and an else for Telemetry 2.0

File: dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/CrossFeatureJaegerTest.java (hunk 3 above)
Comment (Yas-Franco) [reply dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/CrossFeatureJaegerTest.java:157]: We have an if for 1/1.1 and an else for Telemetry 2.0

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Review by benjamin-confino — DISMISSED** [review]

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Review by benjamin-confino — DISMISSED** [review]

**Comment by yasmin-aumeeruddy** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Review by Channyboy — APPROVED** [review]

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#build

**Comment by Yas-Franco** [comment]
The build had an existing defect. 

Successful SOE here: https://libh-proxy1.fyre.ibm.com/cognitive-dev/pipelineAnalysis.html?pipelineId=a3d5cc58-a398-47a5-87de-57ea8d049204&tab=Tests

**Pull request merged**